tests/integration/compose.yaml

```diff
@@ -63,6 +63,8 @@ services:
       IDM_ADMIN_PASSWORD: admin
       PROXY_AUTOPROVISION_ACCOUNTS: true
       PROXY_ROLE_ASSIGNMENT_DRIVER: oidc
+      MICRO_REGISTRY: natsjs
+      MICRO_REGISTRY_ADDRESS: 127.0.0.1:9233
```

Comment:
```suggestion
      MICRO_REGISTRY_ADDRESS: 0.0.0.0:9233
```

Reply:
but it's just for a while. fix should arrive in ocis soon

Reply:
This PR just to check changes in share api

Reply:
may be wait until PR https://github.com/owncloud/ocis/pull/8028 is merged?

Reply:
Ahh, okey

Reply:
we will not merge this PR, we just want to check share api current behavior